widgetastic_manageiq.py

```diff
@@ -1333,3 +1335,171 @@ def all_emails(self):
 
     def read(self):
         return list(self.all_emails)
+
+
+class TimelinesZoomSlider(View):
+    """This control represents Timeline's Zoom Slider
+
+    """
+    ROOT = ParametrizedLocator('{@locator}')
+    zoom_in_button = Text(locator='//button[@id="timeline-pf-zoom-in"]')  # button +
+    zoom_out_button = Text(locator='//button[@id="timeline-pf-zoom-out"]')  # button -
+
+    def __init__(self, parent, locator, logger=None):
+        View.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    @property
+    def value(self):
+        return float(self.browser.get_attribute('value', self))
+
+    @cached_property
+    def max(self):
+        return float(self.browser.get_attribute('max', self))
+
+    @cached_property
+    def min(self):
+        return float(self.browser.get_attribute('min', self))
+
+    def zoom_in(self):
+        self.zoom_in_button.click()
+
+    def zoom_out(self):
+        self.zoom_out_button.click()
+
+    def zoom_max(self):
+        while self.value < self.max:
+            self.zoom_in()
+
+    def zoom_min(self):
+        while self.value > self.min:
+            self.zoom_out()
+
+    def read(self):
+        return self.value
+
+
+class TimelinesFilter(View):
+    """represents Filter Part of Timelines view
+
+    """
+    # common
+    event_type = BootstrapSelect(id='tl_show')
+    event_category = BootstrapSelect(id='tl_category_management')
+    time_period = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+    time_range = BootstrapSelect(id='tl_range')
+    time_position = BootstrapSelect(id='tl_timepivot')
+    # date_picker = Calendar()
```

Comment:
removed this comment
I've to put in more efforts so that this view is fully covered.  I've created only really necessary part elements. The rest will be added later.